staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -144,7 +144,7 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 		"SSL Certificate Authority file used to secure etcd communication.")
 
 	fs.BoolVar(&s.StorageConfig.Quorum, "etcd-quorum-read", s.StorageConfig.Quorum,
```

Comment:
Seems like this should just default to true.  @kubernetes/sig-api-machinery-misc any arguments against?

Reply:
agree. etcd2 was inefficient around q-get. etcd3 is efficient. there is almost no reason to not enable it by default for k8s use case.

Reply:
/cc @jpbetz

Reply:
Agree, default to true. I'd even warn if user sets it to non-true.

Reply:
opened https://github.com/kubernetes/kubernetes/pull/53717 to change the default to true

Reply:
Radical suggestion: lets just remove the flag and always enable it? (or rather, ignore the flag with deprecation warning for compatibility, and always enable quorum reads).

Aiui, quorum-read is equivalently fast in a trivial/test single-etcd setup, and wrong in a clustered-etcd setup - so there's no legitimate case to ever have this false (is that correct?)

Reply:
@anguslees that's exactly what @liggitt did in #53717 :)

Reply:
@dims: no, he just changed the default to true.  I'm suggesting *we remove the ability to make it false*

Reply:
@anguslees aha. got it. thanks!

Reply:
We should mark it deprecated first, right? and then remove the flag (i.e. remove the ability to make it false) after 6 months as per our deprecation policy . open #53795.